graphite-ch-optimizer.go

```diff
@@ -58,11 +58,10 @@ GROUP BY
 	partition_id
 -- modified_at < rollup_time: the merge has not been applied for the current retention policy
 -- parts > 1: merge should be applied because of new parts
--- modified_at < (now() - @Interval): we want to merge active partitions only once an interval
--- @Interval < age: do not touch currently active partitions
+-- modified_at < (now() - @Interval): we want to merge active partitions only once an interval,
```

Comment:
If partitions only get merged once in the duration of an interval, `only once per interval` will make it more clear